lib/core/data.coffee

```diff
@@ -0,0 +1,39 @@
+KDEventEmitter = require './eventemitter'
+
+module.exports = class KDData extends KDEventEmitter
```

Comment:
`Proxy` object is required to achieve observe functionality over `KDData` I don't want to introduce specific setters to use on a `KDData` instance, instead we can use these instances as a native objects/arrays.